yaksh/views.py

```diff
@@ -121,25 +129,68 @@ def user_register(request):
     if request.method == "POST":
         form = UserRegisterForm(request.POST)
         if form.is_valid():
-            u_name, pwd, user_email, key = form.save()
-            new_user = authenticate(username=u_name, password=pwd)
-            login(request, new_user)
-            if user_email and key:
-                success, msg = send_user_mail(user_email, key)
-                context = {'activation_msg': msg}
-                return my_render_to_response(
-                    request,
-                    'yaksh/activation_status.html', context
-                )
-            return index(request)
+            if not settings.IS_DEVELOPMENT:
+                ''' Begin reCAPTCHA validation '''
```

Comment:
It is unusual to write docstrings in the middle of the code. Please using `#` for commenting.